soco/data_structures.py

```diff
@@ -502,6 +506,8 @@ def from_element(cls, element):     # pylint: disable=R0914
             if result is not None:
                 # We store info as unicode internally.
                 content[key] = really_unicode(result)
+            else:
```

Comment:
Isn't this redundant, since `content` turns into kwargs on `__init__` which is already being handled?